mfs_test.go

```diff
@@ -17,17 +17,20 @@ func TestMain(m *testing.M) {
 	for _, f := range files {
 		file, err := os.Open(fmt.Sprintf("./testdata/%s", f))
 		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to open test data file: %v\n", err)
 			os.Exit(1)
```

Comment:
Why is this not a `t.Fatal(err)`?

Reply:
This is before any test runs so we don't have access to a `t` yet.

Reply:
Makes sense. I wasn’t aware of TestMain.